x-pack/legacy/plugins/maps/common/descriptor_types.d.ts

```diff
@@ -4,11 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { IFieldType } from '../../../../../src/plugins/data/common/index_patterns/fields';
+export type EMSXYZSourceType = 'EMS_XYZ';
```

Comment:
I think we should avoid typing string constants like these. Wrappers around primitive types are just more layers to unravel.